stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/ChatInfoSharedGroupsFragment.kt

```diff
@@ -66,6 +68,11 @@ class ChatInfoSharedGroupsFragment : Fragment() {
                 findNavController().navigateSafely(ChatInfoSharedGroupsFragmentDirections.actionOpenChat(it.cid, null))
             }
 
+            (layoutInflater.inflate(R.layout.shared_groups_empty_view, view, false) as TextView).apply {
```

Comment:
This looks a bit nicer with ViewBinding: 

```suggestion
            SharedGroupsEmptyViewBinding.inflate(layoutInflater).root.apply {
```